WalletWasabi/Crypto/CoordinatorSecretKey.cs

```diff
@@ -30,5 +30,8 @@ private CoordinatorSecretKey(Scalar w, Scalar wp, Scalar x0, Scalar x1, Scalar y
 			new CoordinatorParameters(
 				cw: W * Generators.Gw + Wp * Generators.Gwp,
 				i: Generators.GV - (X0 * Generators.Gx0 + X1 * Generators.Gx1 + Ya * Generators.Ga));
+
+		public ScalarVector GetWitness(Scalar t) =>
+			new ScalarVector(W, Wp, X0, X1, t, Ya);
```

Comment:
`t` can be dropped, it's a mistake from my comments that i shared with you (one of many ;-)

Reply:
Thank you. Honestly I didn't check the math, that's why i didn't realized there was a column only with `Inf`.